packages/js/client/src/__tests__/core/plugin-wrapper.spec.ts

```diff
@@ -17,6 +17,7 @@ const defaultPlugins = [
   "wrap://ens/sha3.polywrap.eth",
   "wrap://ens/graph-node.polywrap.eth",
   "wrap://ens/fs.polywrap.eth",
+  "wrap://ens/ipfs-resolver.polywrap.eth",
```

Comment:
Hey @nerfZael would it make sense to remove `ipfs.polywrap.eth` since the only purpose of it was to help us to resolve and now this uri takes care of that?

Reply:
`ipfs-resolver.polywrap.eth` plugin uses the `ipfs.polywrap.eth` plugin to read from IPFS. Without it, it wouldn't be able to resolve.